mmv1/products/documentaiwarehouse/Location.yaml

```diff
@@ -0,0 +1,101 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Location'
+base_url: 'projects/{{project}}/locations/us:initialize'
+self_link: '{{name}}'
```

Comment:
I don't see the reason that `self_link` is needed here.

Reply:
I'd used `self_link` to pass the right format for the GET request. According to you recommendation, I tried to add `skip_read: true` and remove `self_link: true` , but now the test is failing with an unclear reason. One thing I noticed is, if `self_link: true` is absent then the GET request is performed multiple times (4-6 times), and it is structured like this 

```
---[ REQUEST ]---------------------------------------
GET /v1/projects/406397197218?alt=json&prettyPrint=false HTTP/1.1
```
Looks like here it is checking for the existence of the project.

If `self_link: true` is present then after the POST request, the test errors out with the following error

```
error=
  | exit status 1
  | 
  | Error: Provider produced inconsistent result after apply
  | 
  | When applying changes to google_document_ai_warehouse_location.example,
  | provider "provider[\"registry.terraform.io/hashicorp/google\"]" produced an
  | unexpected new value: Root resource was present, but now absent.
  | 
  | This is a bug in the provider, which should be reported in the provider's own
  | issue tracker.
```
I tried to recheck the binary (the .tfrc file) I have used and that seems to be okay. Am I missing something? Do you still recommend I remove the `self_link: true`

Reply:
It looks like more properties are needed.
```
id_format: 'projects/{{project}}/locations/{{location}}'
exclude_import: true
```

Reply:
added this in the latest commit